src/routes/client.js

```diff
@@ -13,7 +13,7 @@ router.post("/", async function (req, res) {
     logger.information(CLIENT_ROUTE_COMPLETED);
 
     if (response?.upi || response?.msg === CLIENT_DETAILS_MISMATCH) {
-      logger.information(`${GENERATED_NUMBER} ${response.upi || response}`);
+      logger.information(`${response.upi || response.msg}`);
```

Comment:
This is kinda confusing. Can we have `response?.msg === CLIENT_DETAILS_MISMATCH` with `response.upi` logged? I think we are better off moving that logic to a separate function and check explicitly.